views/celebrities/celebrities.hbs

```diff
@@ -0,0 +1,19 @@
+<h2>Celebrities</h2>
+
+{{#each celebrities}}
+
+<div>
+    <h3>{{name}}</h3>
+    <p>{{occupation}}</p>
+    <p>{{catchPhrase}}</p>
+    <a href="/celebrities/{{_id}}/edit">Edit Celebrity!</a>
```

Comment:
The route `/celebrities/{{_id}}/edit` still doesn't exist so this `<a>` tag is taking you nowhere!